### PR TITLE
Switch to new prop-types dependency to keep in sync with React v15.5.4 & add StandardJS linting

### PR DESCRIPTION
**Pull request by yamafaktory**

### CODE REVIEW

**Comment by yamafaktory** [comment]
[Fix warnings (c1)](https://trello.com/c/xVMRcwsq/2543-fix-warnings-c1)

File: package.json [hunk 1]
```diff
@@ -13,24 +13,30 @@
...
+    "lint": "standard --verbose | snazzy",
     "build:commonjs": "BABEL_ENV=commonjs babel --quiet src --out-dir lib",
     "build:es": "BABEL_ENV=es babel --quiet src --out-dir es",
     "build": "npm run clean && npm run build:es && npm run build:commonjs",
-    "prepublish": "npm run clean && npm run build"
+    "prepublish": "npm run lint && npm run clean && npm run build"
   },
   "dependencies": {
-    "glamor": "^2.20.20"
+    "glamor": "2.20.24"
```
Comment (dan-lee) [inline package.json]: As this is acts as a library we didn't want to have pinned versions here.

**Review by dan-lee — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -13,24 +13,30 @@
...
+    "babel-preset-es2015": "6.24.1",
+    "babel-preset-react": "6.24.1",
+    "prop-types": "15.5.8",
+    "react": "15.5.4",
+    "snazzy": "7.0.0",
+    "standard": "10.0.2"
   },
   "peerDependencies": {
-    "react": "^15.4.2"
+    "prop-types": "15.5.8",
```
Comment (dan-lee) [inline package.json]: I think `prop-types` can be a hard dependency here.

**Review by dan-lee — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (yamafaktory) [reply package.json]: I'll change it but FYI using a caret is making the bold assumption that any library you rely on will strongly follow semver guidelines, which is sadly not always the case. IMHO is always better to keep it deterministic.

Comment (dan-lee) [reply package.json]: Yeah essentially you're right! But we want to have as less dependencies as possible here. And I am pretty sure all dependencies we have added to `react-view` follow semver.
If that's not the case we should change it to pinned! :)

**Review by dan-lee — APPROVED** [review]

**Pull request merged**